doc/Preboarding_and_Onboarding.rst

```diff
@@ -44,7 +52,7 @@ Onboarding
 ===========
 
 
-The onboarding procedure must be performed after completing the preboarding procedure.  You should be brought automatically to the correct page after completing the preboarding:
+The onboarding procedure must be performed after completing the preboarding procedure. You should be brought automatically to the correct page after completing the preboarding:
 
 - If you are not or you want to redo the onboarding, navigate to *https://<management IP>:28100/#/onboarding* 
 - If you logged out navigate to: *https://<management IP>:28100/#/preboarding/login*
```

Comment:
Need to change to HTTPS after recent code changes.

Reply:
@shailesh-vaidya, Its already defined as HTTPS but seems you wanted to say HTTP as per recent change

Reply:
CORTX UI shifted to 443 port so it will be default HTTPS and `28100` port is not required. e.g. https://3.108.161.44/#/preboarding/welcome

Reply:
@TechWriter-Mayur, this change is requested in https://github.com/Seagate/cortx/pull/1104 which I closing as duplicate so address the change accordingly as suggested